StructureDefinition/England-MedicationDispense.xml

```diff
@@ -0,0 +1,363 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="England-MedicationDispense" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/England-MedicationDispense" />
+  <version value="1.0.0" />
+  <name value="EnglandMedicationDispense" />
+  <title value="England MedicationDispense" />
+  <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="Medication Dispense" />
+  This profile defines the NHS England constraints and extensions on the FHIR UK Core resource [UK Core MedicationDispense](https://simplifier.net/hl7fhirukcorer4/ukcore-MedicationDispense).
+  <purpose value="Medication Dispense" />
```

Comment:
Check the correct wording for purpose element in [UKCore-MedicationDispense](https://simplifier.net/hl7fhirukcorer4/ukcore-medicationdispense)